frontend/views/containers/modals/LeaveGroup.vue

```diff
@@ -10,7 +10,7 @@
 
       i18n(tag='p' html='If you leave, you will stop having access to the <b>group chat</b> and <b>contributions</b>. Re-joining the group is possible, but requires other members to <b>vote and reach an agreement</b>.')
 
-      message(severity='danger')
+      banner-simple(severity='danger')
         i18n(tag='p' html=' This action <b>cannot be undone</b>.')
```

Comment:
Boyscoutting: think you can update these i18n tags to use the [new i18n LTags w/args behavior](https://github.com/okTurtles/group-income-simple/blob/master/frontend/views/utils/translations.js#L20-L27) since `html=` [is deprecated](https://github.com/okTurtles/group-income-simple/blob/master/frontend/views/utils/translations.js#L87-L90)?